core/find_apk.sh

```diff
@@ -8,7 +8,15 @@ getlatestapk() {
   tempapks=( )
   OLDIFS="$IFS"
   IFS="
-" # We set IFS to newline here so that spaces can survive the for loop
+" # We set IFS to newline here so that filenames containing spaces are
+  # handled correctly in the for-loops below.
+
+  # decompress apks
+  # some apks are lz compressed to workaround github file-size limits.
+  for foundapklz in $(find "$1" -iname '*.apk.lz'); do
+    lunzip $foundapklz
```

Comment:
Regarding clean up, since the APK name always changes (since it's equivalent to the APK version number), that shouldn't be a problem.

But, rest of the PR looks good to me, and I'll create a task to clean up extracted .lz files.

Thanks for your contribution!